models/project/board.go

```diff
@@ -150,12 +163,27 @@ func createBoardsForProjectsType(ctx context.Context, project *Project) error {
 	return db.Insert(ctx, boards)
 }
 
+// maxProjectColumns max columns allowed in a project, this should not bigger than 127
+// because sorting is int8 in database
+const maxProjectColumns = 20
```

Comment:
This should have a description in docs.

Reply:
I think you are right but currently, there is no documentation for projects. This could be added after project documentation is added.